src/ContainerLogs.jsx

```diff
@@ -70,7 +70,7 @@ class ContainerLogs extends React.Component {
         this.connectStream();
     }
 
-    componentDidUpdate(prevProps, prevState) {
+    componentDidUpdate(prevProps, _prevState) {
```

Comment:
I get why we have to do the `_` prefix on the start of function arguments. But on the tail end we can just drop them altogether.